src/scopyExceptionHandler.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SCOPYEXCEPTIONHANDLER_H
+#define SCOPYEXCEPTIONHANDLER_H
+
+#include <QMessageBox>
+#include <QApplication>
+#include <config.h>
+#include <libm2k/m2kexceptions.hpp>
+#include <scopyApplication.hpp>
+
+using namespace adiscope;
+
+#define HANDLE_EXCEPTION(e) \
+	if(GetScopyApplicationInstance()->getDebugMode()) { \
+		QMessageBox msg; \
+		QString str; \
```

Comment:
You could ditch the string and in the if else do `setText`

Reply:
I still need to append whether a minidump was created or not.